src/lib/style/base/_core.scss

```diff
@@ -1223,3 +1216,74 @@ hr ~ .SRC-wrapper {
     margin-right: 5px;
   }
 }
+
+$link-underline-color-default: adjust-color(
+  SRC.$primary-action-color,
+  $saturation: -50%,
+  $lightness: 40%
+) !default;
+
+@mixin highlightLink() {
+  border-bottom: 1px solid $link-underline-color-default;
+  &:hover {
+    border-bottom: 2px solid SRC.$primary-action-color-700;
+  }
+}
+
+@mixin normalLink() {
+  color: SRC.$primary-action-color;
+  text-decoration: none;
+  font-weight: 700;
+  border-bottom: 1px solid transparent;
+  letter-spacing: .5px;
+  &:hover {
+    border-bottom: 1px solid $link-underline-color-default;
+  }
+}
+
+@mixin ignoreLink() {
+  color: unset;
+  text-decoration: unset;
+  font-weight: unset;
+  border-bottom: unset;
+  letter-spacing: unset;
+  &:hover {
+    border-bottom: unset;
+  }
+}
+
+// Core link styles across the portal main area.
+// Design would like "body text" to have the special primary link style.
+// Other links should have the normal secondary link style.
+// You can force normal link style by adding the "secondary-links" css class a container of your anchor.
+// Or force the highlighted primary link by adding "highlight-link" css class name to an anchor.
+main, #rootPanel {
+  a:not(.btn), .bootstrap-4-backport a:not(.btn), 
+  .markdown td a:not(.btn),
+  .markdown a.SRC-userCard:not(.btn),
+  .secondary-links a:not(.btn),
+  .markdown .secondary-links a:not(.btn)
+   {
+    @include normalLink;
+    &:visited, &:active, &:link {
+      @include normalLink;
+    }
+  }
+
+  .markdown a, .SRC-userCardName a, a.highlight-link,
+  details>summary {
+    &:not(.btn) {
+      @include highlightLink;
+      &:visited, &:active, &:link {
+        @include highlightLink;
+      }
+    }
+  }
+
+  // do not apply special styling to dropdown links or anchors in tabs
+  .dropdown, .nav-tabs, .dropdown-menu {
+    a:not(.btn) {
+      @include ignoreLink;
+    }
+  }
+}
```

Comment:
Have you checked if the new link style will render correctly for the top menu in mobile view (< 768px)? Thanks.

<img width="722" alt="Screen Shot 2021-07-16 at 2 00 51 PM" src="https://user-images.githubusercontent.com/434792/126008072-1de84b1c-d541-42db-ba1e-461478d091f6.png">

Reply:
Menu looks ok to me.  Thanks for bringing this up though, the style change could certainly have unintended side effects in buried UI like this.

Reply:
<img width="622" alt="Screen Shot 2021-07-16 at 2 17 19 PM" src="https://user-images.githubusercontent.com/1864447/126009548-601a3245-562a-4ed7-9c42-ba241280d760.png">